src/codegen/cfg.rs

```diff
@@ -1027,7 +1029,7 @@ impl ControlFlowGraph {
                 true_block,
                 false_block,
             ),
-            Instr::LoadStorage { ty, res, storage } => format!(
+            Instr::LoadStorage { ty, res, storage, .. } => format!(
```

Comment:
a comment clarifying about how the system behaves when the storage type is omitted might be helpful to people looking at this in the future.

Reply:
This is the implementation of `instr_to_string`, which is used in Solang if a Solidity dev wants to debug the generated IR (using `--emit cfg).
To simplify the output of this, I omitted the storage type option as I think it is not relevant at the IR generation stage.
WDYT?